packages/mock-server/src/api/mutations/invoice.ts

```diff
@@ -49,7 +49,16 @@ export const InvoiceMutation = {
           input.id,
           invoiceNumber,
           otherPartyName,
-          InvoiceNodeType.OutboundShipment
+          InvoiceNodeType.OutboundShipment,
+          {
+            status: InvoiceNodeStatus.New,
+            allocatedDatetime: null,
+            pickedDatetime: null,
+            shippedDatetime: null,
+            deliveredDatetime: null,
+            verifiedDatetime: null,
+            comment: null,
+          }
```

Comment:
This is for creating an outbound. It is utilising the same `createInvoice` as the random data generator, so nulling out all the fields from random values.